cloudbuild_CI.yaml

```diff
@@ -70,6 +49,8 @@ steps:
       - '--run_all_tests'
       - '--test_name_prefix cloud-ci-'
     id: 'test-gcp-variant-transforms-docker'
+    entrypoint: './deploy_and_run_tests.sh'
```

Comment:
The `./` gives me pause.  I'd usually expect to see either the bare script (to allow exec to do PATH lookup) or a full path.

It's not clear to me why this works (eg, this suggests the working directory inside the container contains the script)